src/coreclr/jit/compiler.h

```diff
@@ -2006,14 +2006,16 @@ struct NaturalLoopIterInfo
     // The local that is the induction variable.
     unsigned IterVar = BAD_VAR_NUM;
 
+#ifdef DEBUG
+    // Tree that initializes induction varaible outside the loop.
```

Comment:
```suggestion
    // Tree that initializes induction variable outside the loop.
```

Reply:
Going to address this as part of a follow-up.